### PR TITLE
feat: feedback footer

### PR DESCRIPTION
**Pull request by its-me-dhina**
Feedback footer to docs

### CODE REVIEW

**Review by harshayellow — APPROVED** [review]

File: static/js/FeedbackFooter.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+window.onload = function () {
+  setInterval(() => {
+    renderFeedbackFooter();
+  }, 2000);
+};
```
Comment (subbu3012) [inline static/js/FeedbackFooter.js:5]: setInterval(renderFeedbackFooter, 2000);
};

File: static/js/FeedbackFooter.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+function renderFeedbackFooter() {
+  const parentDOM = document.getElementById("__docusaurus");
+  const container = parentDOM.querySelectorAll(
+    '[class^="docMainContainer"]'
+  )[0];
```
Comment (subbu3012) [inline static/js/FeedbackFooter.js:11]: why is it `.querySelectorAll` and `[0]`?

**Comment by subbu3012** [comment]
and why is package-lock changed?

File: static/js/FeedbackFooter.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  const footer = document.getElementById("feedbackFooter");
+  const feedbackFooterHTML =
+    "<div id='feedbackFooter' style='padding-left: 48px; padding-bottom: 20px;'><h3>Was the article helpful?</h3><a style='padding: 10px; text-decoration: none;' href='#'><i class='ri-thumb-up-fill'></i></a><a style='padding: 10px; text-decoration: none;' href='#'><i class='ri-thumb-down-fill'></i></a></div>";
+  if (container && !footer) container.innerHTML += feedbackFooterHTML;
```
Comment (subbu3012) [inline static/js/FeedbackFooter.js:15]: can you use `appendChild`?

**Review by subbu3012 — COMMENTED** [review]
needs some cleanup.

File: docusaurus.config.js [hunk 4]
```diff
@@ -126,5 +126,6 @@ module.exports = {
   onBrokenLinks: "log",
   scripts: [
     "/js/bot.js", // connect support bot 
+    "/js/FeedbackFooter.js", // feeback footer
```
Comment (subbu3012) [inline docusaurus.config.js:129]: better add `FeedbackFooter.js` before `bot.js`.

**Pull request merged**